core/src/main/scala/org/apache/spark/scheduler/Pool.scala

```diff
@@ -47,6 +49,15 @@ private[spark] class Pool(
   var name = poolName
   var parent: Pool = null
 
+  override def maxShare = {
```

Comment:
Maybe specifying return types? (See Return types in https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide)